### PR TITLE
Add CHARTER.md

### PR DESCRIPTION
**Pull request by jchestershopify**
Adds the charter from the template project, with one small elision (see https://github.com/ossf/project-template/issues/4 for details).

Request to reviewers: please ensure I haven't overlooked any placeholder names / variables.

### CODE REVIEW

**Comment by jchestershopify** [comment]
I caught my fork up with a merge in the Github UI but had expected a rebase. Let me know if that's no good and you'd prefer me to go back and create a rebased PR.

File: CHARTER.md
Comment (di) [inline CHARTER.md:11]:
```suggestion
- a. The mission of the Technical Initiative is to be for and focus on the maintainers of software repositories, software registries, and tools which rely on them, at various levels including system, language, plugin, extensions and container systems. It provides a forum to share experiences and to discuss shared problems, risks and threats. This mission is elaborated in the Technical Initiative's [Objectives statement](https://github.com/ossf/wg-securing-software-repos#objective).
```

**Review by di — COMMENTED** [review]
Otherwise LGTM

**Comment by jchestershopify** [comment]
In discussion in the Securing Critical Projects WG a question was raised about whether the whole TSC mechanism makes sense, and it's unclear whether @ossf/tac will be updating the documentation as part of their work on OpenSSF process docs. I think we might want to hold on merging this for the moment until matters are clearer.

**Review by bobcallaway — COMMENTED** [review]
LGTM

**Comment by trishankatdatadog** [comment]
Cool! WDYT about some academics like @mnm678 to balance it out, like in the [SigStore root](https://github.com/sigstore/root-signing)?

**Comment by jchestershopify** [comment]
I went back and forth on that, actually, and I fell very slightly on the side of limiting it to repo folks and organizers. But I am happy to be argued around to the other point of view. FWIW @mnm678 would easily qualify under the number of attendances criterion.

**Comment by trishankatdatadog** [comment]
> I went back and forth on that, actually, and I fell very slightly on the side of limiting it to repo folks and organizers. But I am happy to be argued around to the other point of view. FWIW @mnm678 would easily qualify under the number of attendances criterion.

Maybe let's vote at the next meeting?

**Comment by jchestershopify** [comment]
Works for me.

File: CHARTER.md
Comment (bobcallaway) [inline CHARTER.md:5]: need to update this upon vote

**Comment by jswank** [comment]
LGTM

**Review by bobcallaway — APPROVED** [review]

**Review by znewman01 — APPROVED** [review]

**Review by MylesBorins — APPROVED** [review]
LGTM

**Review by simi — APPROVED** [review]

**Review by Eh2406 — APPROVED** [review]

**Review by bettymakes — APPROVED** [review]

**Review by jswank — APPROVED** [review]

**Review by caabernathy — APPROVED** [review]

**Review by jswank — APPROVED** [review]

**Review by di — APPROVED** [review]

**Comment by bobcallaway** [comment]
Per the charter document, we need 2/3 approval to establish or amend the charter; I see approvals from the following maintainers:

1. @jchestershopify (implicit as PR author)
2. @di 
3. @bobcallaway 
4. @jswank 
5. @MylesBorins 
6. @Eh2406 
7. @simi 

7 of 9 is sufficient for this vote to pass (@trevrosen and @brianf were not in attendance at the meeting)

File: CHARTER.md
Comment (jchestershopify) [reply CHARTER.md:5]: I'll open a PR momentarily.

**Pull request merged**
